.github/workflows/main.yaml

```diff
@@ -5,18 +5,18 @@ on:
     branches:
       - 'main'
 env:
-  go-version: "1.20"
+  go-version: "1.21"
   filename: 'main.yaml'
 jobs:
   static-check:
     name: Static Check
     runs-on: ubuntu-22.04
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Setup golang
-        uses: actions/setup-go@v3
+        uses: actions/setup-go@v4
         with:
           go-version: ${{ env.go-version }}
```

Comment:
Could you use `go-version-file` instead of env variable?
https://github.com/actions/setup-go/tree/v4#getting-go-version-from-the-gomod-file
https://github.com/cybozu-go/moco/blob/v0.18.1/.github/workflows/e2e.yaml#L20-L21